src/Heap.ts

```
@@ -804,7 +804,9 @@ export class Heap<T> implements Iterable<T> {
    * @param  {Number} k Another node index
    */
   _moveNode(j: number, k: number): void {
-    [this.heapArray[j], this.heapArray[k]] = [this.heapArray[k], this.heapArray[j]];
+    let temp = this.heapArray[j];
```

ESLint

[error] 'temp' is never reassigned. Use 'const' instead.

(prefer-const)